devtools/cli/src/main/java/io/quarkus/cli/config/Encryptor.java

```diff
@@ -0,0 +1,99 @@
+package io.quarkus.cli.config;
+
+import java.nio.ByteBuffer;
+import java.nio.charset.StandardCharsets;
+import java.security.MessageDigest;
+import java.security.SecureRandom;
+import java.util.Base64;
+import java.util.concurrent.Callable;
+
+import javax.crypto.Cipher;
+import javax.crypto.KeyGenerator;
+import javax.crypto.SecretKey;
+import javax.crypto.spec.GCMParameterSpec;
+import javax.crypto.spec.SecretKeySpec;
+
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Option;
+
+@Command(name = "encryptor", aliases = "enc", header = "Encrypt Secrets using AES/GCM/NoPadding algorithm by default")
```

Comment:
Wouldn't it be better to use a verb here? `encrypt` rather than `encryptor`?

Other than that, I think it should go in, except if @maxandersen has additional comments?

Reply:
We can change the name. I can't remember why I've picked that name in the first place :)

Reply:
Ok, renamed.